plugins/woocommerce/src/Enums/ProductStockStatus.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Enums;
+
+/**
+ * Enum class for all the product stock statuses.
+ */
+final class ProductStockStatus {
```

Comment:
We should also update https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Enums/README.md to mention this enum.

Reply:
Did it in https://github.com/woocommerce/woocommerce/pull/54725/commits/231c5148f4a677a23db80759c5112bf4a5caacde